docs/source/api/notebooks/demo_run_train.py

```diff
@@ -0,0 +1,292 @@
+"""
+    Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License").
+    You may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Demonstration models for using GraphStorm APIs
+"""
+
+
+import argparse
+import graphstorm as gs
+import dgl
+import torch as th
+import torch.nn as nn
+import torch.nn.functional as F
+
+from graphstorm.config import get_argument_parser
+from graphstorm.config import GSConfig
+from graphstorm.model.gnn_encoder_base import GraphConvEncoder
+from graphstorm.model import (GSgnnNodeModel,
+                              GSNodeEncoderInputLayer,
+                              RelationalGATEncoder,
+                              EntityClassifier,
+                              ClassifyLossFunc)
+
+
+class Ara_GatLayer(nn.Module):
```

Comment:
Is it possible to have a model.py?

Reply:
Yes, but in order to avoid errors when users copy and paste multiple files, I tried to make a Python file self-contained.